server/sonar-web/src/main/js/apps/issues/filters-view.js

```diff
@@ -75,26 +75,26 @@ export default Marionette.ItemView.extend({
 
   saveAs () {
     const query = this.options.app.controller.getQuery('&');
-    const url = '/issues/save_as_form?' + query;
+    const url = window.baseUrl + '/issues/save_as_form?' + query;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)